modules/shortcodes/email-subscribe.php

```diff
@@ -119,13 +119,13 @@ public function parse_shortcode( $attrs ) {
 		// We allow for overriding the presentation labels.
 		$data = shortcode_atts(
 			array(
-				'title'             => '',
+				'title'             => __( 'Join my email list', 'jetpack' ),
```

Comment:
This was omitted, not sure why. @artpi ?

Reply:
This was discussed here:
https://github.com/Automattic/jetpack/pull/10597#issuecomment-451487056

> > 1. This one could probably go either way, but there seems to be a Gutenberg convention of not building titles in blocks.
>
> good point. I made title an optional parameter and don't include a default one, so its not showing.

Related commit: b825bd1b20b6e7cedb79fe573cfccc6b74c0569e

It consequently seems safe to revert to no default title here.